packages/ozone-typescript-client/src/utility/utility.ts

```diff
@@ -9,7 +9,7 @@ export function returnNullOn404<T>(response: Response<T>) {
 	}
 }
 
-export function deepCopy<T= any>(obj: T): T {
+export function deepCopy<T extends any>(obj: T): T {
```

Comment:
Why it's better to extend any rather than use any as default?

Reply:
Actually I do not know if it is better, but with the `=` I get the following ts error: `Property 'hasOwnProperty' does not exist on type 'T'`
![Screenshot 2019-11-27 at 13 32 18](https://user-images.githubusercontent.com/5877409/69723682-7e0fad80-111a-11ea-9338-aa5e6090d179.png)

Reply:
> Actually I do not know if it is better, but with the `=` I get the following ts error: `Property 'hasOwnProperty' does not exist on type 'T'`
> ![Screenshot 2019-11-27 at 13 32 18](https://user-images.githubusercontent.com/5877409/69723682-7e0fad80-111a-11ea-9338-aa5e6090d179.png)

I see. I think we cloud do this as well  `T extends Object = any`